code/modules/research/techweb/nodes/misc_nodes.dm

```diff
@@ -6,14 +6,15 @@
 	description = "Big Data, in space!"
 	prereq_ids = list("base")
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 1000)
-
+/*
 /datum/techweb_node/adv_datatheory
 	id = "adv_datatheory"
```

Comment:
if you are commenting out a techweb, you gotta fix the ones that need this for requirement